weaknesses/MASVS-STORAGE/MASWE-0004.md

```diff
@@ -21,3 +21,25 @@ status: draft
 
 ---
 
+## Overview
+
+iOS and Android automatically back up app's to the cloud. These files often include sensitive user and app data. Under certain conditions, the backup may not be adequately secured by the cloud provider, or a malicious actor may tamper with the backed-up files. It's a good practice to review how your app instructs the system to perform backups and exclude sensitive files. This page also covers local backups made to your laptop and device-to-device transfers.
```

Comment:
I tried to pull the last sentence into the beginning to improve the flow, and try not to talk about mitigation, instead focusing on the developer's actions or lack of them.

```suggestion
iOS and Android automatically back up app data to cloud services, and users can also create local backups on physical machines, or backups are automatically created during data transfers when switching between phones. When developers fail to properly configure how their app handles backups and neglect to exclude sensitive files, the backups may contain sensitive user and app data. Under certain conditions, the backups may not be adequately secured by the cloud provider, or a malicious actor could tamper with the backed up files, potentially altering the app's behavior or extracting confidential information.
```